contracts/helpers/WETHOmnibridgeRouter.sol

```diff
@@ -0,0 +1,54 @@
+pragma solidity 0.7.5;
+
+import "../interfaces/IOmnibridge.sol";
+import "../interfaces/IWETH.sol";
+import "../libraries/AddressHelper.sol";
+import "../upgradeable_contracts/modules/OwnableModule.sol";
+import "../upgradeable_contracts/Claimable.sol";
+
+contract WETHOmnibridgeRouter is OwnableModule, Claimable {
+    IOmnibridge public immutable bridge;
+    // solhint-disable-next-line var-name-mixedcase
+    IWETH public immutable WETH;
```

Comment:
Does it make sense to note somehow (e.g. by the comments) that this contract is intended to work with particular implementations of wrapped tokens like: 
  * WETH9: https://etherscan.io/address/0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2
  * WBNB: https://bscscan.com/address/0xbb4CdB9CBd36B01bD1cBaEBF2De08d9173bc095c